blog/electron-37-0.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Electron 37.0.0
+date: 2025-06-24T00:00:00.000Z
+authors:
+  - georgexu99
+  - VerteDinde
+slug: electron-37-0
+tags: [release]
+---
+
+Electron 37.0.0 has been released! It includes upgrades to Chromium 138, V8 13.8, and Node 22.16.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 37.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/release?channel=stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Bluesky](https://bsky.app/profile/electronjs.org) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Corner Smoothing is Here: A Design Enhancement Option
+
+![An image showing different corner smoothing values (0%, 30%, 60%, and 100%) applied to rectangles, with 60% labeled as matching macOS style](../static/assets/img/corner-smoothing.svg)
+
+Electron 37 (ok, you got us! This feature technically landed in Electron 36, but we felt like it deserved a larger spotlight) introduces a new CSS feature that brings Apple-inspired smooth corners to your applications. The new `-electron-corner-smoothing` CSS property allows for further design customization closer aligning with the MacOS design language.
```

Comment:
I think I'd put the "ok, you got us" into a separate line (or omit it). It kind of breaks the flow.

Reply:
You could also say: "We backported this feature to Electron 36 to let you use it earlier."